docs/docs/low_level_app.md

```diff
@@ -0,0 +1,159 @@
+---
+id: low_level_app
+title: Low level upgradeable app
+sidebar_label: Low level upgradable app
+---
+
+> **Note**: this guide shows a low-level method for operating a complex upgradeable decentralized application. For a CLI-aided developer experience, use the [higher-level CLI guide](setup.md).
+
+> **Note**: for a fully working project with this example, see the [`examples/complex`](https://github.com/zeppelinos/zos-lib/tree/master/examples/complex) folder of the `zos-lib` repository.
+
+Most real-world applications require more than a single smart contract. In this guide we will explore how to build an upgradeable app with multiple smart contracts and how to use the ZeppelinOS on-chain standard library.
```

Comment:
I'm not sure if that will be a common confusion since there is a big red warning right above telling the user this is not the preferred method, but the CLI is. I'm in with improving it, but if instead we remove it there is no introduction and no difference with the simple lib example.